infra/TEMP-vsts_agent_windows_DEBUG.tf

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) 2019 The DAML Authors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+locals {
+  DEBUG_vsts_token   = "${secret_resource.vsts-token.value}"
+  DEBUG_vsts_account = "digitalasset"
```

Comment:
I’ll never understand where things are called `digital-asset` and where they are called `digitialasset`.